src/app/components/feed/jh-pushtotalk-button.html

```diff
@@ -0,0 +1,10 @@
+<button ng-class="{'btn-danger': vm.hotkeyActive }" class="btn" ng-click="vm.click()" title="{{vm.titleText}}">
+  <span class="glyphicon glyphicon-bullhorn" aria-hidden="true">
+  </span>
+  <span ng-show="vm.showHotkey">
+    <kbd style="background-color:#f15a5c">
+        {{vm.hotkey}}
```

Comment:
Just a nickpicking detail. When the mode is activated and the key is displayed, the header grows a little bit, at least in Firefox.